suite-common/wallet-core/src/accounts/accountsReducer.ts

```diff
@@ -169,98 +165,94 @@ export const selectAccountsByNetworkAndDevice = memoizeWithArgs(
     },
 );
 
-export const selectAccountLabel = memoizeWithArgs(
-    (state: AccountsRootState, accountKey: AccountKey) => {
-        const account = selectAccountByKey(state, accountKey);
-        const accounts = selectAccounts(state);
-
-        const accountData = accounts.find(acc => acc.descriptor === account?.descriptor);
-        if (accountData) {
-            const {
-                metadata: { accountLabel },
-            } = accountData;
-            return accountLabel;
-        }
-    },
-);
-
-export const selectAccountNetworkSymbol = memoizeWithArgs(
-    (state: AccountsRootState, accountKey: AccountKey): NetworkSymbol | null => {
-        const account = selectAccountByKey(state, accountKey);
+export const selectAccountLabel = (state: AccountsRootState, accountKey: AccountKey) => {
+    const account = selectAccountByKey(state, accountKey);
+    const accounts = selectAccounts(state);
 
-        if (!account) return null;
+    const accountData = accounts.find(acc => acc.descriptor === account?.descriptor);
+    if (accountData) {
+        const {
+            metadata: { accountLabel },
+        } = accountData;
+        return accountLabel;
+    }
```

Comment:
I think that at the moment when was @Nodonisko changing this file, my branch was not yet merged. There will be a conflict during rebase and it will be resolved.

Reply:
ha, looking forward. I have changes to this place too in #8646

Reply:
Oh I see, it's not rebased yet 😅

Reply:
Only thing I did it is removal of memoize